region/lowernorfair/east.json

```diff
@@ -1712,49 +2081,85 @@
               "id": 4,
               "strats": [
                 {
-                  "name": "Space Screw",
-                  "notable": false,
-                  "requires": [
-                    "h_canNavigateHeatRooms",
-                    "ScrewAttack",
-                    "SpaceJump",
-                    {"heatFrames": 300}
-                  ],
-                  "obstacles": [
-                    {
-                      "id": "A",
-                      "requires": []
-                    }
-                  ]
-                },
-                {
-                  "name": "Writg IBJ",
+                  "name": "IBJ",
                   "notable": false,
                   "requires": [
                     "h_canIBJ",
-                    "h_heatProof"
+                    {"or": [
+                      {"and": [
+                        "canJumpIntoIBJ",
+                        {"heatFrames": 1460}
+                      ]},
+                      {"and": [
+                        "canDoubleBombJump",
+                        {"heatFrames": 1000}
+                      ]},
+                      {"and": [
+                        "canJumpIntoIBJ",
+                        "canDoubleBombJump",
+                        {"heatFrames": 800}
+                      ]},
+                      {"heatFrames": 2000}
+                    ]}
                   ],
                   "obstacles": [
                     {
                       "id": "A",
-                      "requires": []
+                      "requires": [
+                        {"or": [
+                          "h_canUsePowerBombs",
```

Comment:
does the PB require falling back down, or does it require wall jumping up and placing the PB first?

Reply:
Weird. I didnt think this was possible. Do you know if it is possible to CF in an IBJ? it could allow some higher CF+Grapple clips. We weren't able to get it when we were trying.

Reply:
That is solidly TAS only tech.